### ISSUE TITLE
webpack 4 compatibility issue

### ISSUE THREAD

**Issue by veeramarni** [sysgears/spinjs#1]
webpack 4 favors .mjs over .js files and applies stricter rules when dealing with Harmony modules.

![image](https://user-images.githubusercontent.com/3382355/44005679-436b95ac-9e45-11e8-8d4f-765246389749.png)

The following is what suggested to use. 
```
{
  test: /\.mjs$/,
  include: /node_modules/,
  type: "javascript/auto",
},
```

Should the spinjs handle `.mjs` files as well?

**Comment by vlasenko** [comment]
@veeramarni Yes, I think yes

**Comment by vlasenko** [comment]
@veeramarni Published `spinjs@0.4.151`, it adds the block you have proposed into webpack config by default.
